rhalphalib/model.py

```diff
@@ -363,23 +363,35 @@ def autoMCStats(self, epsilon=0, threshold=0, include_signal=0, channel_name=Non
         first_sample = self._samples[list(self._samples.keys())[0]]
 
         for i in range(first_sample.observable.nbins):
-            ntot, etot2 = 0, 0
+            ntot_bb, etot2_bb = 0, 0  # for the decision to use bblite or not
```

Comment:
In the docstring can you explain that the include signal flag is only to determine whether or not to use bb-lite?

Reply:
Added